src/components/playground/Playground.tsx

```diff
@@ -24,13 +25,14 @@ const useDebouncedValue = (inputValue: string, delay: number) => {
   return debouncedValue
 }
 
-const Playground: React.FC<PlaygroundProps> = ({defaultApiEndpoint}) => {
+const Playground = () => {
   const apiEndpointParam =
     typeof window !== "undefined" && new URLSearchParams(window.location.search).get("apiEndpoint")
   const initialApiEndpoint =
```

Comment:
```suggestion
const Playground = () => {
  const apiEndpointParam =
    typeof window !== "undefined" && new URLSearchParams(window.location.search).get("u")
  const initialApiEndpoint =
```

Rename the param to `u` meaning the URL that we would like to load.